test/classes/CarparkTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+require '/Users/benmorris/Documents/Development/academy/parking-system/classes/Carpark.php';
+require '/Users/benmorris/Documents/Development/academy/parking-system/vendor/autoload.php';
+use PHPUnit\Framework\TestCase;
+
+
+/**
+ * Created by PhpStorm.
+ * User: benmorris
+ * Date: 15/11/2016
+ * Time: 14:17
+ */
+class CarparkTest extends TestCase
+{
+    public function testSuccessfulInstantiation() {
+
+        $mockPDO = $this->createMock('PDO');
+        $mockStatement = $this->createMock('PDOStatement');
+        $mockStatement->method('fetch')->willReturn([
+            'id' => '1',
+            'name' => 'test_name',
+            'capacity' => 'test_capacity',
+            'isVisitor' => 1
+        ]);
+        $mockPDO->method('prepare')->willReturn($mockStatement);
+
```

Comment:
In order to fetch, the prepared statement needs to be executed after having been prepared, so I would imagine the execute method on $mockStatement will needs to appear somewhere in the test.